src/Persistence.LinnApps/Repositories/RequisitionRepository.cs

```diff
@@ -46,7 +46,7 @@ public override async Task<RequisitionHeader> FindByIdAsync(int key)
                 .FirstOrDefaultAsync(r => r.ReqNumber == key);
         }
 
-        public override async Task<IList<RequisitionHeader>> FilterByAsync(
+        public async Task<IList<RequisitionHeader>> FilterByAsync(
```

Comment:
there isn't a FilterByAsync method with this signature on the interface anymore, so you shouldn't have one in the implementations